src/chunk_insert_state.c

```diff
@@ -535,6 +549,12 @@ ts_chunk_insert_state_create(Chunk *chunk, ChunkDispatch *dispatch)
 	return state;
 }
 
+void
+ts_chunk_insert_state_switch(ChunkInsertState *state, ChunkDispatch *dispatch)
+{
+	adjust_slots(state);
```

Comment:
How come this function only exists as a wrapper around `adjust_slots`, which is only called from here? Can we inline adjust slots into this function? (or just remove this function, I think `ts_chunk_insert_state_adjust_slots` is a more evocative name)